config/environments/production.rb

```diff
@@ -19,9 +19,12 @@
   # or in config/master.key. This key is used to decrypt credentials (and other encrypted files).
   # config.require_master_key = true
 
-  # Disable serving static files from the `/public` folder by default since
-  # Apache or NGINX already handles this.
-  config.public_file_server.enabled = ENV['RAILS_SERVE_STATIC_FILES'].present?
+  # Unless explicitly disabled, serve static files via puma. This has been the
+  # default for Dockerfile for a very long time already.
+  config.public_file_server.enabled = (ENV['RAILS_SERVE_STATIC_FILES'] || "true") == "true"
+  config.public_file_server.headers = {
+    'Cache-Control' => "public, max-age=#{1.hour.to_i}"
```

Comment:
The current `nginx.conf` has `public, max-age=2419200, must-revalidate` for most assets, `public, max-age=604800, must-revalidate` for `sw.js`, and `public, max-age=2419200, immutable` for user-uploaded files.

Can we have a similar split?